.gitignore

```diff
@@ -18,3 +18,12 @@ cmd/generate/config/gitleaks.toml
 *.out
 
 dist/
+
+# Hugo build files (created automatically by VSCode Go extension)
+.hugo_build.lock
+
+# Hugo Sitemap.xml files (created automatically by VSCode Go extension)
+public/index.xml
+public/sitemap.xml
+public/categories/index.xml
+public/tags/index.xml
```

Comment:
```suggestion
```

These should probably not be part of the PR.

Reply:
These files are created by `go run main.go`. 

Should these `.xml` files be included in the repository? If they should, then I'll commit them. If not, they should be in the .gitignore. Given that they're automatically built, `.gitignore` seems more appropriate. Yes, it isn't an atomic PR, but they should be there.

Reply:
@dvasdekis Maybe leave this out and get the PR approved? I'm really looking forward for your changes in the next release. :)